com.unity.shadergraph/ShaderGraphLibrary/Functions.hlsl

```diff
@@ -107,4 +107,7 @@ void shadergraph_Fog(float3 position, out float4 color, out float density)
     density = 0;
 }
 
+
+#define SHADERGRAPH_TRANSFORM_TEX(uv, texStruct) uv * texStruct.scaleTranslate.xy + texStruct.scaleTranslate.zw
```

Comment:
might want to move this into texture.hlsl?  It relies on the texture structures in that file.
Might even be good to make this an actual function on those structures, instead of a macro?

Reply:
Texture hlsl is core renderpipeline though, right? Do we want a shadergraph function there? But I agree with making it a function I think

Reply:
changed